frontend/www/js/omegaup/components/contest/Mine.vue

```diff
@@ -32,9 +32,22 @@
               <span>{{ T.contestListShowAdminContests }}</span>
             </label>
           </div>
+          <div class="form-check col-md-3">
+            <label class="form-check-label">
+              <input
+                v-model="shouldShowArchivedContests"
+                class="form-check-input"
+                type="checkbox"
+                @change.prevent="
+                  $emit('show-archived-contests', shouldShowArchivedContests)
```

Comment:
```suggestion
                  $emit('change-show-archived-contests', shouldShowArchivedContests)
```

por consistencia con la otra propiedad.

Reply:
Listo